bindings/python/pinocchio/__init__.py

```diff
@@ -13,3 +13,13 @@
 from .libpinocchio_pywrap import *
 from .deprecated import *
 from .shortcuts import *
+
+if pin.WITH_FCL:
+  try:
+    import hppfcl as fcl
```

Comment:
This may be confusing people with bindings of the original FCL library. I suggest to keep hppfcl.

Reply:
I will change it.

Reply:
Thanks.